#! /bin/sh /usr/share/dpatch/dpatch-run
## 01_DESCRIPTION.dpatch by  <edd@xmcorsairs.wu-wien.ac.at>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Add space Package: and name

@DPATCH@

diff -ru HAPim.orig/DESCRIPTION HAPim/DESCRIPTION
--- HAPim.orig/DESCRIPTION	2008-08-25 15:12:59.000000000 +0200
+++ HAPim/DESCRIPTION	2009-05-12 03:13:29.000000000 +0200
@@ -1,4 +1,4 @@
-Package:HAPim
+Package: HAPim
 Type: Package
 Title:HapIM
 Version: 1.2
